packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts

```diff
@@ -8,6 +8,22 @@ import { IDebuggerMessage } from "./Messages";
 
 // #region Outbound messages
 
+/**
+ * Base interface for telemetry log events.
+ * @public
+ */
+export interface TelemetryLogEvent {
```

Comment:
Let's follow the convention of naming interfaces with an `I` at the beginning, and I suggest this other name.
```suggestion
export interface ITimestampedTelemetryEvent {
```